src/modules/transfer/views/ViewTransferCreate.vue

```diff
@@ -647,15 +649,31 @@ export default {
     let mosaic = account.assets.find(
       (asset) => asset.idHex == selectedMosaic.value[e.index].id
     );
+    // enable back the option
+    for (let i in mosaics.value){
+      mosaics.value[i].disabled = false
+    }
+    // disable all the options choosen
+    mosaics.value.filter(o1 => selectedMosaic.value.some(o2 => {
+      if(o1.val == o2.id){
+        o1.disabled = true;
+      }
+    }));
+
     selectedMosaic.value[e.index].amount = "0";
     mosaicSupplyDivisibility.value[e.index] = mosaic.divisibility;
     emitter.emit("CLOSE_MOSAIC_INSUFFICIENT_ERR", false);
   };
+
   const removeMosaic = (e) => {
-    mosaicsCreated.value.splice(e.index, 1);
+    if(selectedMosaic.value[e.index].id != 0){
```

Comment:
should check the index exist first
`if(selectedMosaic.value[e.index] && selectedMosaic.value[e.index].id){`

Reply:
noted will change it